lib/travis/build/script/r.rb

```diff
@@ -164,6 +165,10 @@ def configure
               options_repos = "options(repos = c(#{repos_str}))"
               sh.cmd %Q{echo '#{options_repos}' > ~/.Rprofile.site}
 
+              # Set -Wall -pedantic in the site Makevars to catch more issues,
+              # as CRAN runs with -pedantic
+              sh.cmd %Q{echo 'PKG_CXXFLAGS=-Wall -pedantic\nPKG_CFLAGS=-Wall -pedantic' > ~/.Makevars.site}
```

Comment:
Shouldn't we then use another environment variable? I'm not sure if user's Makevars are expected to honor existing `PKG_???` values. (Mine don't.)